src/configs/config-variables.ts

```diff
@@ -108,8 +108,7 @@ function handleEnvVars(): void {
     URI: process.env.CONNECT_MANAGEMENT_URL || "",
     APIKey: process.env.CONNECT_MANAGEMENT_API_KEY || "",
   };
-  configVariables.profileMockServerUrl =
-    process.env.PROFILE_MOCK_SERVER_URL || "";
+  configVariables.connectProfileUrl = process.env.PROFILE_MOCK_SERVER_URL || "";
```

Comment:
```suggestion
  configVariables.connectProfileUrl = process.env.PROFILE_MOCK_SERVER_URL || process.env.PROFILE_URL || "";
```
This way, we could just configure `process.env.PROFILE_URL` in the prod environment and make it work in all cases.
Maybe this shouldn't be changed yet though.